cmd/rds.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright © 2023 NAME HERE <EMAIL ADDRESS>
+*/
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/spf13/cobra"
+	"github.com/surajincloud/awsctl/pkg/rds"
+)
+
+// rdsCmd represent the rds command
+var rdsCmd = &cobra.Command{
+	Use:   "rds",
+	Short: "Retrieve information about AWS RDS instances",
+	Long: `This command retrieves information about AWS RDS instances, including their instance ID, status, endpoint, engine and version.
+Example:
+	awsctl get rds`,
+	RunE: getRdsCommand,
+}
+
+func getRdsCommand(cmd *cobra.Command, args []string) error {
+
+	rdsInstances, err := rds.GetRDSInstances()
+
+	if err != nil {
+		return errors.New("failed to retrive RDS instances : " + err.Error())
+	}
+
+	if len(rdsInstances) == 0 {
+		fmt.Println("No RDS instances found")
+		return nil
+	}
+	w := tabwriter.NewWriter(os.Stdout, 5, 2, 3, ' ', tabwriter.TabIndent)
+	defer w.Flush()
+
+	fmt.Fprintln(w, "DB_INSTANCE_ID", "\t", "STATUS", "\t", "ENDPOINT", "\t", "ENGINE", "\t", "VERSION")
+
+	for _, i := range rdsInstances {
+		fmt.Fprintf(w, "%s\t%s\t%s\t%s\t%s\n", i.DBInstanceID, i.Status, i.Endpoint, i.Engine, i.Version)
```

Comment:
can you do the same for this as well, it makes code easier to read.

Reply:
Removed  `fmt.Fprintf(w, "%s\t%s\t%s\t%s\t%s\n", i.DBInstanceID, i.Status, i.Endpoint, i.Engine, i.Version)` and included 
`fmt.Fprintln(
			w,
			i.DBInstanceID, "\t",
			i.Status, "\t",
			i.Endpoint, "\t",
			i.Engine, "\t",
			i.Version, "\t",
		)`